### ISSUE TITLE
Can't use commands from other plugins with the same name

### ISSUE THREAD

**Issue by kerphloo** [Niestrat99/AT-Rewritten#25]
**Is your feature request related to a problem? Please describe.**
If another plugin uses the same commands (EssentialsX in this case), there is no way to use that plugin's commands instead of AT's. If you try to disable `at.member.warp`, it spits out a permissions error. If you try to disable it in the config, it just says "This feature is disabled.".

**Describe the solution you'd like**
When the features are disabled in the config, allow other plugins to use those commands. Possibly have a boolean to choose whether it shows the error message or not.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Thatsmusic99** [comment]
I'm afraid that this feature on AT's end may not be possible, at all. Until further notice, I will label it with a "wontfix".

Your only real workaround is by using the commands.yml file provided by Bukkit, which exists for this very reason; to tackle command conflicts. For example, if you want EssentialsX to handle /warp instead of AT, add the following alias to the commands file:

```yml
aliases:
  warp: 
  - essentials:warp $1-```

For more information, please refer to https://bukkit.gamepedia.com/Commands.yml .

Thank you for your time!

**Comment by Thatsmusic99** [comment]
Addressed in [v5.4-SNAPSHOT-3](https://github.com/Niestrat99/AT-Rewritten/releases/tag/v5.4-SNAPSHOT-3)
